From e7c552ec897894ec421867059e48474eb7f1ff6d Mon Sep 17 00:00:00 2001
From: "Michael S. Tsirkin" <mst@redhat.com>
Date: Fri, 25 Feb 2022 06:46:34 -0500
Subject: [PATCH] virtio: drop default for virtio-mem
Git-commit: e7c552ec897894ec421867059e48474eb7f1ff6d
Patch-mainline: v5.17-rc8
References: bsn#198
Modified-by-SEL: No


There's no special reason why virtio-mem needs a default that's
different from what kconfig provides, any more than e.g. virtio blk.

Signed-off-by: Michael S. Tsirkin <mst@redhat.com>
Acked-by: David Hildenbrand <david@redhat.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 drivers/virtio/Kconfig |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/virtio/Kconfig
+++ b/drivers/virtio/Kconfig
@@ -105,7 +105,6 @@ config VIRTIO_BALLOON
 
 config VIRTIO_MEM
 	tristate "Virtio mem driver"
-	default m
 	depends on X86_64
 	depends on VIRTIO
 	depends on MEMORY_HOTPLUG_SPARSE
